include/ttmlir/Target/TTNN/program.fbs

```diff
@@ -36,7 +36,8 @@ enum EltwiseOpType: uint32 {
   Relu = 3,
   GreaterEqual = 4,
   Sqrt = 5,
-  Div = 6
+  Div = 6,
+  Reciprocal = 7,
```

Comment:
Heads up: there's gonna be conflicts with other added ops

Reply:
Yep, I know, waiting for you to merge, I will rebase. :)